### ISSUE TITLE
Change time for interval-based tax collection from "minimally 48 hours" to "maximally 1 hour"

### ISSUE THREAD

**Issue by Azzurite** [D3nnis3n/White-Tiger-Feedback#27]
I read the discussion in https://github.com/D3nnis3n/White-Tiger-Feedback/issues/8

And it seems like the solution was to increase the tax timer to 48 hours minimum.

This however still leaves the same problems. You still want to do the actions described in the issue as close to the tax tick as possible, leaving players that can play close to the tax tick at an advantage.

The other way makes much more sense: make taxes based on intervals have a **maximum** interval of 1 hour! This way, it's basically impossible to game the system for anyone. You can still theoretically "time" your buys for the hour, but the tax rate will be so much less (i.e. a 20% 48-hour tax would now roughly be a 0.5% hourly tax) that it's basically not worth doing it, especially with the time that it would cost each time. And there's no real way to regain that much money directly after to make it worth it.

So yes, I propose to change the interval from "minimally 48 hours" to "maximally 1 hour" intervals. I wish to do a referendum for that immediately on current White Tiger as well.

**Comment by DanielKispert** [comment]
Take another example:

48h cycle:
Player A sells 20k worth of products to Player B 12h before the tax tick. Player B now has 12 hours to login and spend that money. They also only have to login once every 2 days to close their store if they do not want any extra sales to them.

1h cycle:
Player A sells 20k worth of products to Player B. Until Player B logs in, he will now eat a hourly wealth tax. This leads to Player B not putting up any buy orders so people do not sell to him.

Furthermore, people that can play more have a big benefit because within an hour or two you can easily fulfill some buy orders and then buy new materials with the money.
A transaction based tax like the "store usage tax" that taxes trade profit has none of these drawbacks. It does not favor buy orders over sale orders (something we direly need), it does not care if a person is online, you cannot dodge it and gains are weighed like losses.

**Comment by Azzurite** [comment]
You probably meant "Player A buys" not "Player A sells", otherwise your example doesn't make sense. (Player B would have lost money, so he wouldn't have to "spend that money"). I assume that's what you meant.

Now look at it from this perspective:

48h cycle:
Player A buys 20k worth of products from Player B 10 seconds before the tax tick. Player B now has 10 seconds to login and spend that money, otherwise a huge chunk of that 20k is immediately going to go away through a huge tax tick.

1h cycle:
Player A buys 20k worth of products from Player B. Until Player B logs in, he will now eat a hourly wealth tax, which fortunately is pretty small, since it's hourly, so he only loses roughly 1/48th of the amount for the first tick compared to a 48hour tick. Only if he keeps logged out for 47 more hours is when he loses the same amount that he would have lost in the other scenario, which gives him a lot more time to react.

In other words, the longer you make the tax tick, the more important it is to be online right at the moment of the tax tick. The shorter it is, the less important timing becomes.

**Comment by DanielKispert** [comment]
with the exception that you can just close your store until the tax hits if you don't want to get hit by tax. The longer you make the tax tick, the better it is for players that cannot invest every living hour into WT.

**Comment by Azzurite** [comment]
> with the exception that you can just close your store until the tax hits if you don't want to get hit by tax. The longer you make the tax tick, the better it is for players that cannot invest every living hour into WT.

Is this an argument? This just simply applies to both scenarios in the same way. If you don't open your store for the hourly tax, you also will not get bought out and also will not start paying a lot of taxes.

**Comment by DanielKispert** [comment]
for a player that is online 24/7 this is an option, yes. Not for a player that is online less, then he can at least have some time where the store is open instead of continuously closed.

Shorter tax ticks also favor fulfilling buy orders over putting up your own sale order (like I said). This goes directly against how a supply economy can work (because it needs as many items as possible being up for sale passively) and just makes the "hey can you craft me an X, noone is selling them" problem worse - something we need to avoid at all cost.

**Comment by Azzurite** [comment]
That is true. But that can almost be said about anything - "the player that is online more will have an advantage". I think the benefits of a shorter tax tick greatly outweigh the disadvantages in this case.

**Comment by D3nnis3n** [comment]
I don't think we'll ever come to a fitting solution here, both options have pros and cons and none of them is better than the other, they just have other issues, so the only thing that makes sense is to remove the regulation completely and keep it to the senate and have people sue their asses. The regulation is lifted completely.
